modules/test/playwright/pages/object/modelBuilder.page.ts

```diff
@@ -33,9 +38,14 @@ export class ModelBuilderPage {
 		this.saveNewObjectRelationshipButton = page.getByRole('button', {
 			name: 'Save',
 		});
+		this.toggleSidebarsButton = page.getByLabel('Toggle Sidebars');
+	}
+
+	async clickFitViewButton() {
```

Comment:
@cgmonte just create a new method only if:
1 - It's called outside the POM
2 - It's a complex call and used more than once inside the POM.